docs/src/general/guide-for-latex-users.md

```diff
@@ -0,0 +1,581 @@
+command to set how your document is supposed to look. In that command, you may
+have replaced `article` with another value such as `report` and `amsart` to
+customize your document.
+
+When using Typst, you style your documents with [functions]($type/function).
+Typically, you use a template that provides a function that styles your whole
+document. First, you import the function from a template file. Then, you apply
+it to your whole document. This is accomplished with an "everything" [`{show}`
+rule]($styling/#show-rules), which wraps the following document in a given
+function. An example shows you how it works:
+
+```example:single
+>>> #let conf(
+>>>   title: none,
+>>>   authors: (),
+>>>   abstract: [],
+>>>   doc,
+>>> ) = {
+>>>  set text(font: "Linux Libertine", 11pt)
+>>>  set par(justify: true)
+>>>  set page(
+>>>    "us-letter",
+>>>    margin: auto,
+>>>    header: align(
+>>>      right + horizon,
+>>>      title
+>>>    ),
+>>>    numbering: "1",
+>>>  )
+>>>
+>>>  show heading.where(
+>>>    level: 1
+>>>  ): it => block(
+>>>    align(center,
+>>>      text(
+>>>        13pt,
+>>>        weight: "regular",
+>>>        smallcaps(it.body),
+>>>      )
+>>>    ),
+>>>  )
+>>>  show heading.where(
+>>>    level: 2
+>>>  ): it => box(
+>>>    text(
+>>>      11pt,
+>>>      weight: "regular",
+>>>      style: "italic",
+>>>      it.body + [.],
+>>>    )
+>>>  )
+>>>
+>>>  set align(center)
+>>>  text(17pt, title)
+>>>
+>>>  let count = calc.min(authors.len(), 3)
+>>>  grid(
+>>>    columns: (1fr,) * count,
+>>>    row-gutter: 24pt,
+>>>    ..authors.map(author => [
+>>>      #author.name \
+>>>      #author.affiliation \
+>>>      #link("mailto:" + author.email)
+>>>    ]),
+>>>  )
+>>>
+>>>  par(justify: false)[
+>>>    *Abstract* \
+>>>    #abstract
+>>>  ]
+>>>
+>>>  set align(left)
+>>>  columns(2, doc)
+>>>}
+<<< #import "conf.typ": conf
+#show: conf.with(
+  title: [
+    Towards Improved Modelling
+  ],
+  authors: (
+    (
+      name: "Theresa Tungsten",
+      affiliation: "Artos Institute",
+      email: "tung@artos.edu",
+    ),
+    (
+      name: "Eugene Deklan",
+      affiliation: "Honduras State",
+      email: "e.deklan@hstate.hn",
+    ),
+  ),
+  abstract: lorem(80),
+)
+
+Let's get started writing this article by
+putting insightful paragraphs right here!
+```
+
+The [`{import}`]($scripting/#modules) statement makes
+[functions]($type/function) from another file available. In this example, it
+imports the `conf` function from the `conf.typ` file. The function can format
+content as a conference article. We use the show rule to apply it to the
+document and pass some metadata about the article to the `conf` function.
+Finally, we can get started writing our article below!
+
+<div class="info-box">
+
+Functions are Typst's "commands" and can transform their
+arguments to an output value, including document _content._ Functions can't
+generally manipulate anything they did not receive as an argument.
```

Comment:
Multiple negations, I would simplify to: _Functions can generally manipulate only what they receive as an argument._

What does generally refer to? Are there exceptions? If there are, this would be a natural place to put a comment. Without followup this sentence seems unnecessary.